core/src/main/java/io/grpc/internal/ServerCallImpl.java

```diff
@@ -139,6 +140,8 @@ public void sendMessage(RespT message) {
     } catch (Throwable t) {
       close(Status.fromThrowable(t), new Metadata());
       throw new RuntimeException(t);
+    } finally {
+      GrpcUtil.closeQuietly(resp);
```

Comment:
if resp != null

Reply:
Guava's `Closeables` allows the input stream to be null. Moving the check into `GrpcUtil`